crates/telio-nurse/src/rtt/ping.rs

```diff
@@ -147,7 +140,7 @@ impl Ping {
                 }
                 Err(e) => {
                     results.unsuccessful_pings += 1;
-                    telio_log_error!("Ping error: {}", e.to_string());
+                    telio_log_trace!("Ping {} error: {}", host, e.to_string());
```

Comment:
maybe `telio_log_debug!()` would be better, as this would mean, that builds provided to app teams will have these compiled in, and may help debugging. 

`telio_log_trace!()` is not compiled-in in the above scenario

Reply:
This info is already provided in the [DualPing results log](https://github.com/NordSecurity/libtelio/pull/352/files#diff-5f3177de9978ef3efc6f9a22d22bba482cd008c98e79b921c2b7188268973fd3R65). The question is if the app teams would benefit from knowing that the icmp request failed due to timeout or due to missing route to the host, that would be the only additional info provided by this log.

Reply:
Changed back to debug log level.